src/views/ops/Adhoc/QuickJob.vue

```diff
@@ -232,7 +232,7 @@ export default {
         fold: {
           timeout: {
             type: 'select',
-            name: this.$t('Timeout'),
+            name: this.$t('Timeout(s)'),
             align: 'left',
             value: 60,
             options: [
```

Comment:
There is no syntax error, but it would be more clear if the time unit was explicitly mentioned after "sixty seconds", like so:

```json
export default {
    ...
    timeout: { // Time period in X units  
        ... 
    },
}
```

For a potential tip to optimize this code could be mentioning the use of constants for common values (time), which will help maintain consistency and improve readability:

```javascript
const DEFAULT_TIMEOUT = 60;
const MAX_TIME_TO_WAIT_RECIPE_REQUESTS = Constants.MAX_TIME;

// Use these instead of variable names where necessary:
...
```